src/plugins/myItems/plugin.js

```diff
@@ -20,30 +20,12 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-define([], function () {
-    return {
-        name: "platform/features/my-items",
-        definition: {
-            "name": "My Items",
-            "description": "Defines a root named My Items",
-            "extensions": {
-                "roots": [
-                    {
-                        "id": "mine"
-                    }
-                ],
-                "models": [
-                    {
-                        "id": "mine",
-                        "model": {
-                            "name": "My Items",
-                            "type": "folder",
-                            "composition": [],
-                            "location": "ROOT"
-                        }
-                    }
-                ]
-            }
-        }
+import myItemsIdentifier from "./myItemsIdentifier";
+import myItemsInterceptor from "./myItemsInterceptor";
+
+export default function MyItemsPlugin() {
```

Comment:
Although we don't _really_ need it, if we were to optionally support specifying a namespace (instead of hard-coding the default namespace) we could probably make this a lot more flexible.

Reply:
It would mean that we need a function to be imported, instead of the `myItemsIdentifier` object literal.

Reply:
K, modified it so you can add a namespace, defaults to ''